common/src/main/java/dev/isxander/yacl3/gui/YACLScreen.java

```diff
@@ -84,6 +88,33 @@ protected void init() {
         config.initConsumer().accept(this);
     }
 
+    public void addColorPickerWidget(ColorPickerElement colorPickerElement) {
```

Comment:
Not necessarily for this PR, but this makes me wonder if we can abstract this so it can also be used for e.g. the dropdown controllers, so they can also gain mouse support

Reply:
I think that's possible, yeah. Could probably rename the variables used from "colorPickerVariable" to something like "currentPopupVariable", and then the method would likely accept a class which extends the Abstract Widget or Controller Widget class.